docs/source/tools/non-repudiation.rst

```diff
@@ -0,0 +1,78 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Non-repudiation
```

Comment:
```suggestion
Non-Repudiation
```
the DAML

Reply:
Are you sure? I don't think you are supposed to capitalize hyphenated names (see https://en.wikipedia.org/wiki/Non-repudiation).

Reply:
oh you’re right, nvm